polling_stations/apps/data_finder/helpers.py

```diff
@@ -399,9 +399,15 @@ def get_ors_route(self, longlat_from, longlat_to):
 
         return self.Directions(walk_time, walk_dist, ps)
 
+    def get_base_google_url(self):
+        return "{base}&key={key}".format(
```

Comment:
Yep - you can call with `&key=&foo=bar` in the URL and it is the same as not passing `key`